client/package.json

```diff
@@ -9,11 +9,15 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@hookform/resolvers": "^3.3.4",
     "axios": "^1.6.8",
     "next": "^14.2.3",
     "next-themes": "^0.3.0",
+    "nextjs-toploader": "^1.6.12",
     "react": "^18",
-    "react-dom": "^18"
+    "react-dom": "^18",
+    "react-hook-form": "^7.51.4",
+    "zod": "^3.23.8"
```

Comment:
Write custom method using JS functions to validate input/form. 
Don't use library, we only need library if something hard to implement like animations, etc.

This library makes our build bigger, hope you understand.